ydb/core/engine/minikql/minikql_engine_host.cpp

```diff
@@ -908,14 +908,13 @@ ui64 TEngineHost::GetTableSchemaVersion(const TTableId&) const {
     return 0;
 }
 
-ui64 TEngineHost::LocalTableId(const TTableId& tableId) const {
+ui64 LocalTableId(const TTableId& tableId) {
+    // TODO: May be move to <ydb/core/scheme/scheme_tabledefs.h> ?
     return tableId.PathId.LocalPathId;
```

Comment:
May be move to <ydb/core/scheme/scheme_tabledefs.h> ?

Reply:
You cannot replace this method like that, because LocalTableId is NOT a LocalPathId. I think this is just a default implementation for local minikql transactions, where table ids are resolved into `(tabletId, localTableId)` pairs. Having that behavior baked into a public free function doesn't make sense, this is the exception, not the norm.